lib/routes.jsx

```diff
@@ -155,7 +159,18 @@ function buildRoutes() {
 
       localeURLs.push(newkey);
     });
+  });
 
+  MAKER_PARTY_LOCALES.forEach((locale) => {
+    localeURLs.push(`${locale}`, `${locale}/events`, `${locale}/events/resources`);
+    routes.push(
+      <Route key={locale} path={locale} component={require('../components/page.jsx')}>
+	<IndexRedirect to="/" />
+	<Route path="events" component={require('../pages/events.jsx')}>
+	  <Route path="resources" component={require('../pages/event-resources.jsx')} />
```

Comment:
We already talked about this, but just leaving it here as a reminder.

There is funkiness in components/page.jsx and needs <Route path="resources" component={require('../pages/event-resources.jsx')} /> to be a direct child.

R+ with that done.